todolist/parser.go

```diff
@@ -109,14 +110,21 @@ func (p *Parser) parseArbitraryDate(_date string, pivot time.Time) string {
 
 func (p *Parser) parseArbitraryDateWithYear(_date string, year int) time.Time {
 	res := strings.Join([]string{_date, strconv.Itoa(year)}, " ")
-	if date, err := time.Parse("Jan 2 2006", res); err == nil {
+	var date time.Time
+	var err1 error
+	var err2 error
+	if date, err1 = time.Parse("Jan 2 2006", res); err1 == nil {
 		return date
 	}
 
-	if date, err := time.Parse("2 Jan 2006", res); err == nil {
+	if date, err2 = time.Parse("2 Jan 2006", res); err2 == nil {
 		return date
 	}
-	panic(fmt.Errorf("Could not parse the date you gave me: '%s'", _date))
+	fmt.Printf("Could not parse the date you gave me: %s\n", _date)
+	fmt.Println(err1)
+	fmt.Println(err2)
```

Comment:
OK, that's the message that really helps users.
So I will just remove the both error dumps and add these helpers.